experimental/templates/multiplatform-template/androidApp/build.gradle.kts

```diff
@@ -0,0 +1,33 @@
+plugins {
+    kotlin("multiplatform")
+    id("com.android.application")
+    id("org.jetbrains.compose")
+}
+
+kotlin {
+    android()
+    sourceSets {
+        val androidMain by getting {
+            dependencies {
+                implementation(project(":shared"))
+                implementation("androidx.appcompat:appcompat:1.5.1")
+                implementation("androidx.activity:activity-compose:1.6.1")
+            }
+        }
+    }
+}
+
+android {
+    compileSdk = 33
+    defaultConfig {
+        applicationId = "com.myapplication.MyApplication"
+        minSdk = 24
+        targetSdk = 33
+        versionCode = 1
+        versionName = "1.0"
+    }
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
```

Comment:
We have `sourceCompatibility = JavaVersion.VERSION_11` in the old template

Reply:
Same. KMM plugin wizard uses 1.8. I think that a lower version is better if it works.

Reply:
changed to 11